backend/test/users_test.js

```diff
@@ -623,40 +623,41 @@ describe("Tests add admin user", function() {
-      ["ECSE 428", "Software engineering in practice", "ECSE"]
-    );
-    await connection.query(
-      `INSERT INTO course_offerings (id, scheduled_time, semester, course_code, section) VALUES (?,?,?,?,?) ON DUPLICATE KEY UPDATE id=id;`,
-      [321123, "MWF", "winter", "ECSE 428", "001"]
-    );
-    await connection.query(
-      `INSERT INTO users (username,email, password) VALUES (?,?,?) ON DUPLICATE KEY UPDATE username=username;`,
-      ["testUser", "email@domain.com", 12345678]
-    );
-    await connection.query(
-      `INSERT INTO students (student_id,username) VALUES (?,?) ON DUPLICATE KEY UPDATE student_id=student_id;`,
-      [123321123, "testUser"]
-    );
-    await connection.query(
-      `INSERT INTO student_course_offerings (student_id,offering_id,semester) VALUES (?,?,?);`,
-      [123321123, 321123, "winter"]
-    );
-    return users.getStudentData(123321123).then(function(res) {
-      let found = false;
-      let searchingFor = {
-        major: [],
-        minor: [],
-        courses: [{ course_code: "ECSE 428", semester: "winter" }]
-      };
-      if (JSON.stringify(res) == JSON.stringify(searchingFor)) {
-        found = true;
-      }
-      assert(true, found);
-    });
-  });
-});
+// TODO: Rewrite test for retrieving student data
+// describe("Test retrieve student data", function() {
+//   it("responds with valid", async function() {
+//     connection = await mysql.getNewConnection();
+//     await connection.query(
+//       `INSERT INTO courses (course_code,title, department) VALUES (?,?,?) ON DUPLICATE KEY UPDATE course_code=course_code;`,
+//       ["ECSE 428", "Software engineering in practice", "ECSE"]
+//     );
+//     await connection.query(
+//       `INSERT INTO course_offerings (id, scheduled_time, semester, course_code, section) VALUES (?,?,?,?,?) ON DUPLICATE KEY UPDATE id=id;`,
+//       [321123, "MWF", "winter", "ECSE 428", "001"]
+//     );
+//     await connection.query(
+//       `INSERT INTO users (username,email, password) VALUES (?,?,?) ON DUPLICATE KEY UPDATE username=username;`,
+//       ["testUser", "email@domain.com", 12345678]
+//     );
+//     await connection.query(
+//       `INSERT INTO students (student_id,username) VALUES (?,?) ON DUPLICATE KEY UPDATE student_id=student_id;`,
+//       [123321123, "testUser"]
+//     );
+//     await connection.query(
+//       `INSERT INTO student_course_offerings (student_id,offering_id,semester) VALUES (?,?,?);`,
+//       [123321123, 321123, "winter"]
+//     );
+//     return users.getStudentData(123321123).then(function(res) {
+//       let found = false;
+//       let searchingFor = {
+//         major: [],
+//         minor: [],
+//         courses: [{ course_code: "ECSE 428", semester: "winter" }]
+//       };
+//       if (JSON.stringify(res) == JSON.stringify(searchingFor)) {
+//         found = true;
+//       }
+//       assert(true, found);
+//     });
+//   });
```

Comment:
are there any operations for removing the test data inserted ?

Reply:
No this test is outdated hence why I commented it out. Whoever takes the task for testing this endpoint will have to rewrite most of it.